code/modules/awaymissions/maploader/writer.dm

```diff
@@ -54,96 +38,110 @@
-	template_buffer_text = jointext(template_buffer,"")
+
+	template_buffer_text = jointext(template_buffer, "")
 	log_debug("Reading turfs took [stop_watch(timer)]s.")
 
 	if(templates.len == 0)
 		CRASH("No templates found!")
-	var/key_length = round/*floor*/(log(letter_digits.len,templates.len-1)+1)
+
+	var/key_length = round/*floor*/(log(letter_digits.len, templates.len - 1) + 1)
 	var/list/keys[templates.len]
+
 	// Write the list of key/model pairs to the file
 	timer = start_watch()
 	log_debug("Writing out key/model pairs to file header...")
 	var/list/key_models = list()
 	for(var/key_pos in 1 to templates.len)
-		keys[key_pos] = get_model_key(key_pos,key_length)
+		keys[key_pos] = get_model_key(key_pos, key_length)
 		key_models += "\"[keys[key_pos]]\" = ([templates[key_pos]])\n"
 		CHECK_TICK
+
 	dmm_text += jointext(key_models,"")
 	log_debug("Writing key/model pairs complete, took [stop_watch(timer)]s.")
 
 	var/z_level = 0
 	// Loop over all z in our zone
 	timer = start_watch()
 	log_debug("Writing out key map...")
+
 	var/list/key_map = list()
-	for(var/z_pos=1;TRUE;z_pos=findtext(template_buffer_text,".",z_pos)+1)
-		if(z_pos>=length(template_buffer_text))	break
-		if(z_level)	key_map += "\n"
+	for(var/z_pos = 1; TRUE; z_pos = findtext(template_buffer_text, ".", z_pos) + 1)
+		if(z_pos >= length(template_buffer_text))
+			break
+
+		if(z_level)
+			key_map += "\n"
+
 		key_map += "\n(1,1,[++z_level]) = {\"\n"
-		var/z_block = copytext(template_buffer_text,z_pos,findtext(template_buffer_text,".",z_pos))
-		for(var/y_pos=1;TRUE;y_pos=findtext(z_block,";",y_pos)+1)
-			if(y_pos>=length(z_block))	break
-			var/y_block = copytext(z_block,y_pos,findtext(z_block,";",y_pos))
+		var/z_block = copytext(template_buffer_text, z_pos, findtext(template_buffer_text, ".", z_pos))
+		for(var/y_pos = 1; TRUE; y_pos = findtext(z_block, ";", y_pos) + 1)
+			if(y_pos >= length(z_block))
+				break
+
+			var/y_block = copytext(z_block, y_pos, findtext(z_block, ";", y_pos))
 			// A row of keys
-			for(var/x_pos=1;TRUE;x_pos=findtext(y_block,",",x_pos)+1)
-				if(x_pos>=length(y_block))	break
-				var/x_block = copytext(y_block,x_pos,findtext(y_block,",",x_pos))
+			for(var/x_pos = 1; TRUE; x_pos = findtext(y_block, ",", x_pos) + 1)
+				if(x_pos >= length(y_block))
+					break
+
+				var/x_block = copytext(y_block, x_pos, findtext(y_block, ",", x_pos))
 				var/key_number = text2num(x_block)
 				var/temp_key = keys[key_number]
 				key_map += temp_key
 				CHECK_TICK
 			key_map += "\n"
 		key_map += "\"}"
-	dmm_text += jointext(key_map,"")
+
+	dmm_text += jointext(key_map, "")
 	log_debug("Writing key map complete, took [stop_watch(timer)]s.")
 	log_debug("TOTAL TIME: [stop_watch(total_timer)]s.")
+
 	return dmm_text
 
-/dmm_suite/proc/make_template(var/turf/model as turf, var/flags as num)
-	var/use_json = 0
-	if(flags & DMM_USE_JSON)
-		use_json = 1
+/datum/dmm_suite/proc/make_template(turf/model, flags = 0)
+	var/use_json = flags & DMM_USE_JSON
```

Comment:
`use_json` will now have the value of `DMM_USE_JSON` or 0. It will work but it's best to make this a boolean value just for consistency